src/command/root-command/command-log.ts

```diff
@@ -31,45 +30,34 @@ export class CommandLog {
   public divider: (char?: string) => void
 
   constructor(verbosityLevel: VerbosityLevel) {
-    const emptyFunction = () => {
-      return
-    }
-    const divider = (char = '-') => {
-      console.log(char.repeat(process.stdout.columns))
-    }
-    const error = (message: string, ...args: unknown[]) => console.log(bold().white().bgRed(message), ...args)
-    const log = (message: string, ...args: unknown[]) => console.log(message, ...args)
-    const info = (message: string, ...args: unknown[]) => console.log(italic().dim(message), ...args)
-    const dimFunction = (message: string, ...args: unknown[]) => console.log(dim(message), ...args)
-
     switch (verbosityLevel) {
       case VerbosityLevel.Verbose:
-        this.error = error
-        this.quiet = emptyFunction
-        this.verbose = log
-        this.log = log
-        this.info = info
-        this.dim = dimFunction
-        this.divider = divider
+        this.error = Printer.error
+        this.quiet = Printer.emptyFunction
+        this.verbose = Printer.log
+        this.log = Printer.log
+        this.info = Printer.info
+        this.dim = Printer.dimFunction
+        this.divider = Printer.divider
         break
       case VerbosityLevel.Normal:
-        this.error = error
-        this.quiet = emptyFunction
-        this.verbose = emptyFunction
-        this.log = log
-        this.info = info
-        this.divider = divider
-        this.dim = emptyFunction
+        this.error = Printer.error
+        this.quiet = Printer.emptyFunction
+        this.verbose = Printer.emptyFunction
+        this.log = Printer.log
+        this.info = Printer.info
+        this.divider = Printer.divider
```

Comment:
In the `Verbose` and `default` blocks the `dim` comes before the `divider` but here it's the other way.